.github/build.in.yml

```
@@ -541,8 +541,8 @@ jobs:
       - id: PR
         shell: bash
         run: |
-          echo "number=${{ steps.PR_from_push.outputs.number || github.event.pull_request.number }}" >> $GITHUB_OUTPUT
-          echo "pr_labels=${{ steps.PR_from_push.outputs.pr_labels || join(github.event.pull_request.labels.*.name, ',') }}" >> $GITHUB_OUTPUT
+          echo "number=${{ steps.PR_from_push.outputs.number || github.event.pull_request.number }}" >> "$GITHUB_OUTPUT"
+          echo "pr_labels=${{ steps.PR_from_push.outputs.pr_labels || join(github.event.pull_request.labels.*.name, ',') }}" >> "$GITHUB_OUTPUT"
```

```suggestion
          echo "number=${{ steps.PR_from_push.outputs.number || github.event.pull_request.number }}" | tee -a "$GITHUB_OUTPUT"
          echo "pr_labels=${{ steps.PR_from_push.outputs.pr_labels || join(github.event.pull_request.labels.*.name, ',') }}" | tee -a "$GITHUB_OUTPUT"
```
I know that this was pre-existing, but it might still be useful to know what `pr_labels` looks like!